lib/openstack_api/openstack_query.py

```diff
@@ -298,3 +304,104 @@ def parse_and_output_table(
             output = self.generate_table(output, get_html)
 
         return output
+
+    @dataclass
+    class NonExistentCheckParams:
```

Comment:
This dataclass should go into it's own file, in this case it could sit in the `lib/openstack_api` module since it's an implementation detail not shared with the actions layer